cmds/cleanup.go

```diff
@@ -110,61 +110,61 @@ func cleanUpKubernetesResources(c *k8sclient.Client, ns string, selector labels.
 
 	err := deleteDeployments(c, ns, selector)
 	if err != nil {
-		util.Fatalf("%s", err)
+		util.Fatalf("%s\n", err)
 	}
 
 	err = deleteReplicationControllers(c, ns, selector)
 	if err != nil {
-		util.Fatalf("%s", err)
+		util.Fatalf("%s\n", err)
 	}
 
 	err = deleteReplicaSets(c, ns, selector)
 	if err != nil {
-		util.Fatalf("%s", err)
+		util.Fatalf("%s\n", err)
 	}
 
 	err = deleteServices(c, ns, selector)
 	if err != nil {
-		util.Fatalf("%s", err)
+		util.Fatalf("%s\n", err)
 	}
 
 	err = deleteSecrets(c, ns, selector)
 	if err != nil {
-		util.Fatalf("%s", err)
+		util.Fatalf("%s\n", err)
 	}
 
 	err = deleteIngress(c, ns, selector)
 	if err != nil {
-		util.Fatalf("%s", err)
+		util.Warnf("%s\n", err)
 	}
 
 	err = deleteConfigMaps(c, ns, selector)
 	if err != nil {
-		util.Fatalf("%s", err)
+		util.Fatalf("%s\n", err)
 	}
 
 	catalogSelector, err := unversioned.LabelSelectorAsSelector(&unversioned.LabelSelector{MatchLabels: map[string]string{"kind": "catalog"}})
 	if err != nil {
-		util.Fatalf("%s", err)
+		util.Fatalf("%s\n", err)
 	} else {
 		err = deleteConfigMaps(c, ns, catalogSelector)
 		if err != nil {
-			util.Fatalf("%s", err)
+			util.Fatalf("%s\n", err)
 		}
 	}
 
 	err = deleteServiceAccounts(c, ns, selector)
 	if err != nil {
-		util.Fatalf("%s", err)
+		util.Fatalf("%s\n", err)
 	}
 
 	err = deleteEnvironments(c, selector)
 	if err != nil {
-		util.Fatalf("%s", err)
+		util.Warnf("%s\n", err)
```

Comment:
yeah - on openshift we should always list projects first instead of namespaces (users typically don't have karma on openshift to list namespaces)